examples/head_or_tails.py

```diff
@@ -0,0 +1,31 @@
+import random
+
+from nextcord.ext import commands
+
+bot = commands.Bot(command_prefix="h!")
```

Comment:
I would suggest '$' as a prefix for consistency

Reply:
Okay! I added some fixes tho.